<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.5">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2.83">
    <parentIds>
      <string>38</string>
    </parentIds>
    <id>39</id>
    <startId>10</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@2.40">
      <startTime>1603288755893</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@2.40">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@2.23">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.support.steps.input.InputStepExecution.doAbort(InputStepExecution.java:231)</trace>
          <trace>org.jenkinsci.plugins.workflow.support.steps.input.InputStepExecution$1.run(InputStepExecution.java:104)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions>
          <java.io.IOException>
            <detailMessage>Cannot run program &quot;docker&quot;: error=13, Permission denied</detailMessage>
            <cause class="java.io.IOException">
              <detailMessage>error=13, Permission denied</detailMessage>
              <stackTrace>
                <trace>java.lang.UNIXProcess.forkAndExec(Native Method)</trace>
                <trace>java.lang.UNIXProcess.&lt;init&gt;(UNIXProcess.java:247)</trace>
                <trace>java.lang.ProcessImpl.start(ProcessImpl.java:134)</trace>
                <trace>java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)</trace>
                <trace>hudson.Proc$LocalProc.&lt;init&gt;(Proc.java:252)</trace>
                <trace>hudson.Proc$LocalProc.&lt;init&gt;(Proc.java:221)</trace>
                <trace>hudson.Launcher$LocalLauncher.launch(Launcher.java:996)</trace>
                <trace>hudson.Launcher$ProcStarter.start(Launcher.java:508)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.launch(DockerClient.java:301)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.launch(DockerClient.java:282)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.launch(DockerClient.java:279)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.stop(DockerClient.java:174)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.WithContainerStep.destroy(WithContainerStep.java:109)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.WithContainerStep.access$400(WithContainerStep.java:76)</trace>
                <trace>org.jenkinsci.plugins.docker.workflow.WithContainerStep$Callback.finished(WithContainerStep.java:391)</trace>
                <trace>org.jenkinsci.plugins.workflow.steps.BodyExecutionCallback$TailCall.onFailure(BodyExecutionCallback.java:128)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsBodyExecution$FailureAdapter.receive(CpsBodyExecution.java:361)</trace>
                <trace>com.cloudbees.groovy.cps.impl.ThrowBlock$1.receive(ThrowBlock.java:68)</trace>
                <trace>com.cloudbees.groovy.cps.impl.ConstantBlock.eval(ConstantBlock.java:21)</trace>
                <trace>com.cloudbees.groovy.cps.Next.step(Next.java:83)</trace>
                <trace>com.cloudbees.groovy.cps.Continuable$1.call(Continuable.java:174)</trace>
                <trace>com.cloudbees.groovy.cps.Continuable$1.call(Continuable.java:163)</trace>
                <trace>org.codehaus.groovy.runtime.GroovyCategorySupport$ThreadCategoryInfo.use(GroovyCategorySupport.java:129)</trace>
                <trace>org.codehaus.groovy.runtime.GroovyCategorySupport.use(GroovyCategorySupport.java:268)</trace>
                <trace>com.cloudbees.groovy.cps.Continuable.run0(Continuable.java:163)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.SandboxContinuable.access$001(SandboxContinuable.java:18)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.SandboxContinuable.run0(SandboxContinuable.java:51)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsThread.runNextChunk(CpsThread.java:185)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup.run(CpsThreadGroup.java:400)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup.access$400(CpsThreadGroup.java:96)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$2.call(CpsThreadGroup.java:312)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$2.call(CpsThreadGroup.java:276)</trace>
                <trace>org.jenkinsci.plugins.workflow.cps.CpsVmExecutorService$2.call(CpsVmExecutorService.java:67)</trace>
                <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
                <trace>hudson.remoting.SingleLaneExecutorService$1.run(SingleLaneExecutorService.java:136)</trace>
                <trace>jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)</trace>
                <trace>jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)</trace>
                <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
                <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
                <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
                <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
                <trace>java.lang.Thread.run(Thread.java:748)</trace>
              </stackTrace>
              <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
                <c class="list"/>
                <list reference="../c"/>
              </suppressedExceptions>
            </cause>
            <stackTrace>
              <trace>java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)</trace>
              <trace>hudson.Proc$LocalProc.&lt;init&gt;(Proc.java:252)</trace>
              <trace>hudson.Proc$LocalProc.&lt;init&gt;(Proc.java:221)</trace>
              <trace>hudson.Launcher$LocalLauncher.launch(Launcher.java:996)</trace>
              <trace>hudson.Launcher$ProcStarter.start(Launcher.java:508)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.launch(DockerClient.java:301)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.launch(DockerClient.java:282)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.launch(DockerClient.java:279)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.client.DockerClient.stop(DockerClient.java:174)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.WithContainerStep.destroy(WithContainerStep.java:109)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.WithContainerStep.access$400(WithContainerStep.java:76)</trace>
              <trace>org.jenkinsci.plugins.docker.workflow.WithContainerStep$Callback.finished(WithContainerStep.java:391)</trace>
              <trace>org.jenkinsci.plugins.workflow.steps.BodyExecutionCallback$TailCall.onFailure(BodyExecutionCallback.java:128)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsBodyExecution$FailureAdapter.receive(CpsBodyExecution.java:361)</trace>
              <trace>com.cloudbees.groovy.cps.impl.ThrowBlock$1.receive(ThrowBlock.java:68)</trace>
              <trace>com.cloudbees.groovy.cps.impl.ConstantBlock.eval(ConstantBlock.java:21)</trace>
              <trace>com.cloudbees.groovy.cps.Next.step(Next.java:83)</trace>
              <trace>com.cloudbees.groovy.cps.Continuable$1.call(Continuable.java:174)</trace>
              <trace>com.cloudbees.groovy.cps.Continuable$1.call(Continuable.java:163)</trace>
              <trace>org.codehaus.groovy.runtime.GroovyCategorySupport$ThreadCategoryInfo.use(GroovyCategorySupport.java:129)</trace>
              <trace>org.codehaus.groovy.runtime.GroovyCategorySupport.use(GroovyCategorySupport.java:268)</trace>
              <trace>com.cloudbees.groovy.cps.Continuable.run0(Continuable.java:163)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.SandboxContinuable.access$001(SandboxContinuable.java:18)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.SandboxContinuable.run0(SandboxContinuable.java:51)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsThread.runNextChunk(CpsThread.java:185)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup.run(CpsThreadGroup.java:400)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup.access$400(CpsThreadGroup.java:96)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$2.call(CpsThreadGroup.java:312)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$2.call(CpsThreadGroup.java:276)</trace>
              <trace>org.jenkinsci.plugins.workflow.cps.CpsVmExecutorService$2.call(CpsVmExecutorService.java:67)</trace>
              <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
              <trace>hudson.remoting.SingleLaneExecutorService$1.run(SingleLaneExecutorService.java:136)</trace>
              <trace>jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)</trace>
              <trace>jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)</trace>
              <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
              <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
              <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
              <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
              <trace>java.lang.Thread.run(Thread.java:748)</trace>
            </stackTrace>
            <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" reference="../cause/suppressedExceptions"/>
          </java.io.IOException>
        </suppressedExceptions>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <org.jenkinsci.plugins.workflow.support.steps.input.Rejection plugin="pipeline-input-step@2.12">
              <userName>SYSTEM</userName>
              <timestamp>1603288754265</timestamp>
            </org.jenkinsci.plugins.workflow.support.steps.input.Rejection>
          </a>
        </causes>
        <actualInterruption>true</actualInterruption>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>